tutorials/server/install-an-email-server/install-an-email-server.md

```diff
@@ -0,0 +1,422 @@
+---
+id: tutorial-email-server
+summary: Learn how to set up postfix and dovecot in order to host an imap or pop3 email server.
+categories: server
+tags: tutorial, email, dovecot, postfix, server, guide, imap, pop3
+difficulty: 4
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2017-12-17
+author: Varun Patel <varun-patel@live.com>
+
+---
+
+# Install A VPN Server Using Postfix and Dovecot
```

Comment:
surely this should be 'email server'. Also, we don't use Title case on tiles, so, Install an email server with Postfix and Dovecot